src/echo-app/src/renderer/src/plugin-settings-page.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useEffect, useState } from 'react'
+
+import {
+  ECHO_PLUGIN_CONFIG,
+  EchoPluginConfig,
+  EchoPluginConfigs,
+  EchoPluginHook
+} from 'echo-common'
+import * as path from 'path'
+import { importDevPlugin } from './dev-plugins'
+
+function filterPluginsByEnv(pluginConfigs: EchoPluginConfigs) {
+  return import.meta.env.MODE === 'development'
+    ? Object.values(pluginConfigs).filter((config) => config.name.endsWith('-dev'))
```

Comment:
This can probably be `Object.values(pluginConfigs).filter((config) => (import.meta.env.MODE === 'development') === config.name.endsWith('-dev'))`

Reply:
Ok, updated